hack/ginkgo-e2e.sh

```diff
@@ -56,6 +59,7 @@ else
     prepare-e2e
 
     detect-master >/dev/null
+    KUBE_MASTER_URL="https://${KUBE_MASTER_IP:-}" 
```

Comment:
nothing in this branch of the if statement would set KUBE_MASTER_URL.  But, sure, why not.